bitnami/minio/Chart.yaml

```diff
@@ -19,4 +19,4 @@ name: minio
 sources:
   - https://github.com/bitnami/bitnami-docker-minio
   - https://min.io
-version: 4.0.2
+version: 4.0.3
```

Comment:
To any reviewers, not sure if this is a correct new version for the chart as the values.yaml has some incompatible changes. Any suggestions?

Reply:
This change should definitely bump the major or at least the minor.

Reply:
Bumped it to 5.0.0